src/plugins/plugin_integration/public/index.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
For brand new files, this license from 12-29 is not needed. I always just leave empty and let es-lint handle it for me.

Reply:
+1

Reply:
Thanks! Fixed in latest commit